Kiosk/App/Models/Artwork.swift

```diff
@@ -30,13 +46,14 @@ public class Artwork: JSONAble {
         return defaultImages?.first ?? self.images?.first
     }()
 
-    init(id: String, dateString: String, title: String, titleAndDate: NSAttributedString, price: String, date: String) {
+    init(id: String, dateString: String, title: String, titleAndDate: NSAttributedString, price: String, date: String, sold: String) {
```

Comment:
just to play dumb here, but I've been doing a bit of that there swift lately, and it would automatically create these full init functions for me ( I was using structs though ) - can it work for this?